ush/config.sh.RRFS_AK_dev1

```diff
@@ -28,6 +28,8 @@ PREEXISTING_DIR_METHOD="rename"
 PREDEF_GRID_NAME=RRFS_AK_3km
 
 DO_DACYCLE="true"
+#DO_RETRO="true"
+#LBCS_ICS_ONLY="true"
```

Comment:
We will use the retro functions a a lot soon. This is not just a comment. This is a place hold for retro because ti is much easier and safe to uncomment those options for retrospective runs.